src/test/java/com/google/cloud/genomics/dataflow/pipelines/CountReadsITCase.java

```diff
@@ -44,22 +53,63 @@ public void voidEnsureEnvVar() {
   public void testLocal() throws Exception {
     final String OUTPUT = TEST_OUTPUT_GCS_FOLDER + "CountReadsITCase-testLocal-output.txt";
     String[] ARGS = {
+        "--apiKey=" + API_KEY,
+        "--project=genomics-pipelines",
```

Comment:
the project should also be in an environment variable, sorry I didn't notice that last time around